src/main/java/service/TransactionParserImpl.java

```diff
@@ -0,0 +1,34 @@
+package service;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.Transaction;
+
+public class TransactionParserImpl implements TransactionParser {
+    public static final String COMMA = ",";
+    public static final int OPERATION_TYPE_INDEX = 0;
+    public static final int FRUIT_TYPE_INDEX = 1;
+    public static final int AMOUNT_INDEX = 2;
+
+    @Override
+    public List<Transaction> parse(List<String> lines) {
+        if (lines.size() <= 1) {
+            throw new RuntimeException("No operations available in list " + lines);
+        }
+        List<Transaction> transactions = new ArrayList<>();
+        for (int i = 1; i < lines.size(); i++) {
```

Comment:
```suggestion
        for (int i = OFFSET; i < lines.size(); i++) {
```
let's make it more visible